src/editor/components/LessonPlanNotes.js

```diff
@@ -1,14 +1,32 @@
-import React from 'react';
-import { Text, SafeAreaView, StyleSheet } from 'react-native';
-import LessonPlanTextField from './LessonPlanTextField';
+import React, { useState } from 'react';
+import { Text, SafeAreaView, StyleSheet, TextInput } from 'react-native';
+import { useDispatch } from 'react-redux';
+import { addToNote } from '../../services/editor/lessonPlanSlice';
+import store from '../../services/configureStore';
+
+const LessonPlanNotes = ({ sectionType, subtitle }) => {
+  const [sectionContent, setSectionContent] = useState();
```

Comment:
You should pass in a default state like an empty string
```suggestion
  const [sectionContent, setSectionContent] = useState('');
```